packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -53,9 +97,41 @@ export function SquadPageHeader({
           <h1 className="text-center font-bold typo-title2 laptopL:text-left">
             {squad.name}
           </h1>
-          <h2 className="mt-1 text-center text-text-tertiary typo-body tablet:mt-2 laptopL:text-left">
-            @{squad.handle}
-          </h2>
+          <div className="mt-1 flex flex-row items-center justify-center text-text-quaternary tablet:mt-2 laptopL:justify-start">
+            <h2 className="text-center text-text-tertiary typo-footnote laptopL:text-left">
+              @{squad.handle}
+            </h2>
+            <Separator />
+            {createdAt && (
+              <span className="typo-caption2">Created {createdAt}</span>
+            )}
+          </div>
+          <div className="mt-4 flex flex-row items-center gap-2">
+            <Button
+              icon={buttonIcon}
+              size={ButtonSize.Small}
+              variant={
+                isFeatured ? ButtonVariant.Secondary : ButtonVariant.Float
+              }
+              className={
+                isFeatured
+                  ? 'relative border-overlay-primary-cabbage bg-overlay-tertiary-cabbage'
+                  : ''
+              }
+            >
+              {buttonCopy} Squad
+              {isFeatured && (
+                <>
+                  <SparkleIcon className="absolute -top-2.5 right-0" />
+                  <SparkleIcon className="absolute -bottom-2.5 left-0" />
+                </>
+              )}
+            </Button>
+            {/* As mentioned in the JIRA ticket, the value would be replaced in a different PR */}
```

Comment:
could u add this as a todo with the ticket nr that adds this if we have the ticket?

Reply:
Sure, I was looking at the board for the exact ticket, maybe it was in the backlog. Let me check.

Reply:
there are few in the backlog, not sure if all of them are created though, maybe best to check with @DragosIorgulescu

Reply:
Yeah, I checked and it seems to be not there yet. Though let me add the todo.